pkg/kev/config/svcext_test.go

```diff
@@ -85,88 +82,157 @@ var _ = Describe("Service Extension", func() {
 		})
 
 		Describe("validations", func() {
-			Context("with missing workload", func() {
-				JustBeforeEach(func() {
-					svc.Extensions = map[string]interface{}{
-						"x-k8s": map[string]interface{}{
-							"bananas": 1,
-						},
-					}
-
-					parsedK8sCfg, err = config.SvcK8sConfigFromCompose(&svc)
-					Expect(err).NotTo(HaveOccurred())
-				})
-
-				It("returns defaults", func() {
-					defaultWorkload := config.DefaultSvcK8sConfig().Workload
-					Expect(parsedK8sCfg.Workload).To(BeEquivalentTo(defaultWorkload))
-				})
-			})
-
-			Context("invalid/empty workload", func() {
-				JustBeforeEach(func() {
-					svc.Extensions = map[string]interface{}{
-						"x-k8s": map[string]interface{}{
-							"workload": map[string]interface{}{
+			Context("from svc to k8s ext", func() {
+				Context("with missing workload", func() {
+					BeforeEach(func() {
+						svc.Extensions = map[string]interface{}{
+							"x-k8s": map[string]interface{}{
 								"bananas": 1,
 							},
-						},
-					}
+						}
+					})
 
-					parsedK8sCfg, err = config.SvcK8sConfigFromCompose(&svc)
-					Expect(err).NotTo(HaveOccurred())
+					It("returns defaults", func() {
+						defaultWorkload := config.DefaultSvcK8sConfig().Workload
+						Expect(parsedK8sCfg.Workload).To(BeEquivalentTo(defaultWorkload))
+					})
 				})
 
-				When("workload is invalid", func() {
-					It("it is ignored and returns defaults", func() {
-						Expect(parsedK8sCfg).To(BeEquivalentTo(config.DefaultSvcK8sConfig()))
+				Context("invalid/empty workload", func() {
```

Comment:
tiny nitpick, it'd be nice to keep the same wording as in previous context: `with invalid/empty workload configuration`. Should we tweak other context titles below in line with it it?